### ISSUE TITLE
The Community anchor link slide under the search bar

### ISSUE THREAD

**Issue by RusimbiP** [reactjs/reactjs.org#3121]
There is not enough space between the search input field and the community anchor link.
# Steps to reproduce

- visit: [reactjs.org](https://reactjs.org/community/support.html)

# Expected result
All anchor links should be clear

# Actual result
The Community anchor link slide under the search input field
![Screenshot from 2020-07-20 00-19-05](https://user-images.githubusercontent.com/54870069/87886976-61b1df80-ca21-11ea-99bd-8ab16fe7a990.png)

**Comment by rickhanlonii** [comment]
I can't repo, it seems browser/OS specific. Do you want to submit a fix @RusimbiP?

**Comment by RusimbiP** [comment]
Even on Mozilla, it is like that. I am using ubuntu 20. Sure, I would like to submit a fix.

**Comment by RusimbiP** [comment]
Thank you for the clarification as well. I can confirm that it is Os specific because it looks normal on Windows(just checked)

**Comment by rickhanlonii** [comment]
Let me know if you have any troubles getting the website running locally, I think you'll definitely have issue on windows because of https://github.com/reactjs/reactjs.org/issues/3012

**Comment by RusimbiP** [comment]
Sure thanks!!

**Comment by RusimbiP** [comment]
Hi @rickhanlonii I am getting this error when I run `yarn dev`

```
ERROR #98123  WEBPACK

Generating SSR bundle failed

Cannot find module '@babel/compat-data/corejs3-shipped-proposals'
Require stack:
- /home/winner/Documents/github/reactjs.org/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js
- /home/winner/Documents/github/reactjs.org/node_modules/@babel/preset-env/lib/index.js
- /home/winner/Documents/github/reactjs.org/node_modules/@babel/core/lib/config/files/plugins.js
- /home/winner/Documents/github/reactjs.org/node_modules/@babel/core/lib/config/files/index.js
- /home/winner/Documents/github/reactjs.org/node_modules/@babel/core/lib/index.js
- /home/winner/Documents/github/reactjs.org/node_modules/babel-loader/lib/index.js
- /home/winner/Documents/github/reactjs.org/node_modules/gatsby/dist/utils/babel-loader.js
- /home/winner/Documents/github/reactjs.org/node_modules/loader-runner/lib/loadLoader.js
- /home/winner/Documents/github/reactjs.org/node_modules/loader-runner/lib/LoaderRunner.js
- /home/winner/Documents/github/reactjs.org/node_modules/webpack/lib/NormalModule.js
- /home/winner/Documents/github/reactjs.org/node_modules/webpack/lib/NormalModuleFactory.js
- /home/winner/Documents/github/reactjs.org/node_modules/webpack/lib/Compiler.js
- /home/winner/Documents/github/reactjs.org/node_modules/webpack/lib/webpack.js
- /home/winner/Documents/github/reactjs.org/node_modules/gatsby/dist/commands/develop.js
- /home/winner/Documents/github/reactjs.org/node_modules/gatsby-cli/lib/create-cli.js
- /home/winner/Documents/github/reactjs.org/node_modules/gatsby-cli/lib/index.js
- /home/winner/Documents/github/reactjs.org/node_modules/gatsby/dist/bin/gatsby.js

File: .cache/develop-static-entry.js
```

**Comment by sijusamson** [comment]
@RusimbiP are you working on it ? If not I can open a PR for it.

**Comment by RusimbiP** [comment]
Thanks @sijusamson. I just managed to run it locally! The work begins now.

**Comment by sijusamson** [comment]
@RusimbiP All the best. :-)

**Comment by TechSaq** [comment]
Has this issue been assigned to someone? I would like to do it.
Please assign it to me if none is assigned.

**Comment by RusimbiP** [comment]
I have a PR open

**Comment by rickhanlonii** [comment]
Fixed by @RusimbiP in #3134.

@RusimbiP how did you end up getting past the error you posted?

**Comment by RusimbiP** [comment]
I did a `yarn reset` and re-cloned the project.
